plugins/splitbutton/plugin.js

```diff
@@ -21,10 +21,19 @@
 			face = definition.face,
 			i,
 			item,
+			itemName,
 			previousButton,
 			defaultButton;
 
 		if ( face ) {
+			// Split Button can be defined with simpler definition via strings, needed by #2091
+			if ( typeof face === 'string' ) {
+				face = editor.ui.items[ face ];
```

Comment:
You assigning an object that you modify few lines below. That means you're modifying referenced icon.

We can not do that, you should clone it first and work on it clone instead.